langchain/retrievers/document_compressors/chain_extract.py

```diff
@@ -69,9 +69,10 @@ def from_llm(
         llm: BaseLanguageModel,
         prompt: Optional[PromptTemplate] = None,
         get_input: Optional[Callable[[str, Document], str]] = None,
+        **llm_chain_kwargs: dict[str, Any]
```

Comment:
this should be `Dict[str, Any]` (`dict` is only doable with newer python versions